app/src/test/java/com/appunite/loudius/domain/UserLocalDataSourceTest.kt

```diff
@@ -27,21 +26,25 @@ import strikt.assertions.isEmpty
 import strikt.assertions.isEqualTo
 
 class UserLocalDataSourceTest {
-    private val sharedPreferences = FakeSharedPreferences()
```

Comment:
**info** I couldn't create fake implementation for EncryptedSharePrefrences so I used mocks.